content/en/intelligent_test_runner/troubleshooting/_index.md

```diff
@@ -28,7 +28,17 @@ Due to these restrictions, the first time you enable Intelligent Test Runner, yo
 
 Intelligent Test Runner only takes into account the commit history and test code coverage information for the past month. Additionally, it does not take into account code coverage information that is generated more than one week after a commit was made.
 
-There is a limitation when [synchronizing a fork through GitHub's UI][4] which causes all tests to be run for the generated synchronization commit.
+### Synchronizing a fork through GitHub's UI
+
+[Synchronizing a fork through GitHub's UI][4] causes all tests to be run for the generated synchronization commit.
+
+### Collecting coverages on GitHub Actions CI triggered by pull request events
+
+If you run tests on GitHub Actions CI using the [`pull_request` trigger][5] tests might not be skipped. Code coverage information collected from merge commits is not supported.
+
+### Squash and merge your commits
+
+Using [squash and merge][6] to integrate commits into de main branch causes the git history of the feature branch to be lost. As a result Intelligent Test Runner can run tests in the main branch that should have been skipped.
```

Comment:
`run tests in the main branch that should have been skipped ` - I think we recommend set main branch as excluded so that tests are not skipped. Mentioning that tests in main might not be skipped could cause confusion.

Reply:
I've replaced `main` with `base`, so it's generic and doesn't apply to `main` necessarily.